verifier/verifier.go

```diff
@@ -89,18 +89,18 @@ func (v *verifier) SkipVerify(ctx context.Context, artifactRef string) (bool, *t
 // successful verification.
 // If nil signature is present and the verification level is not 'skip',
 // an error will be returned.
-func (v *verifier) Verify(ctx context.Context, desc ocispec.Descriptor, signature []byte, opts notation.VerifyOptions) (*notation.VerificationOutcome, error) {
+func (v *verifier) Verify(ctx context.Context, desc ocispec.Descriptor, signature []byte, opts notation.VerifierVerifyOptions) (*notation.VerificationOutcome, error) {
 	artifactRef := opts.ArtifactReference
 	envelopeMediaType := opts.SignatureMediaType
 	pluginConfig := opts.PluginConfig
 	logger := log.GetLogger(ctx)
 
-	logger.Debugf("Verify signature against artifact %v referenced as %s in signature media type %v", desc.Digest, artifactRef, opts.SignatureMediaType)
+	logger.Debugf("Verify signature against artifact %v referenced as %s in signature media type %v", desc.Digest, artifactRef, envelopeMediaType)
 	trustPolicy, err := v.trustPolicyDoc.GetApplicableTrustPolicy(artifactRef)
 	if err != nil {
 		return nil, notation.ErrorNoApplicableTrustPolicy{Msg: err.Error()}
 	}
-	logger.Debugf("Trust policy configuration: %+v", trustPolicy)
+	logger.Infof("Trust policy configuration: %+v", trustPolicy)
```

Comment:
Shouldn't this be debug else it will get printed on console?

Reply:
Resolving as we had an offline discussion on it.